### DISCUSSION TITLE
Running lung tutorial

### DISCUSSION THREAD

**Discussion by halim-k** [Chaste/Chaste#63] [Q&A] [answered]
I prepared the environment for running the tutorial by following the instruction, 
I got this message down below after running './TestAirwayGenerationTutorial'.

***** TestAirwayGenerationTutorial.hpp ***** 

Entering TestGenerateAirways
Passed
OK! 

I expected the output will be generated in the 'testoutput' directory, 
but I couldn't find them anywhere, I checked the input files, and I installed VTK 8.2 
I'm wondering if I need to implement some functions to use the code.

I'm not familiar with using C++, would I get some tips for solving this issue?

**Comment by mirams** [comment, marked as answer]
The testoutput normally appears in `/tmp/$USER/testoutput/` - is there anything in there?
